### PR TITLE
chore(data): add a machine learning project and idea

### PR DESCRIPTION
**Pull request by Ashborn-SM**

### CODE REVIEW

File: data/ideas.json [hunk 1]
```diff
@@ -28,5 +28,11 @@
         "description": "Code up a clone of the windows calculator. It’s great practice for GUIs and basic math operation. And if it helps you solve your algebra homework - all the better!",
         "level": "basic",
         "tags": "calculator"
+    },
+    {
+        "title": "Feature maps and Filter Visualizer",
+        "description": "This repository is about the code to extract cnn layers and visualise the filters and featuremaps",
```
Comment (ming-tsai) [inline data/ideas.json:34]: What do you mean about `cnn`?

File: data/ideas.json [hunk 2]
```diff
@@ -28,5 +28,11 @@
...
+        "level": "basic",
+        "tags": "python, machine learning"
```
Comment (ming-tsai) [inline data/ideas.json:36]: The first tag is as idea's id, and is using for match with the project
```suggestion
        "tags": "feature-maps-and-filter-visualizer, python, machine learning"
```

File: data/ideas.json (hunk 1 above)
Comment (Ashborn-SM) [reply data/ideas.json:34]: convolutional neural network

Comment (ming-tsai) [reply data/ideas.json:34]: could you change `cnn` to `convolutional neural network`?

Comment (Ashborn-SM) [reply data/ideas.json:34]: Consider it done

Comment (ming-tsai) [reply data/ideas.json:34]: Did you commit and push it?

File: data/ideas.json (hunk 2 above)
Comment (ming-tsai) [reply data/ideas.json:36]: Please apply the suggestion @Ashborn-SM

File: data/projects.json [hunk 3]
```diff
@@ -98,5 +98,15 @@
...
+    },
+    {
+        "name": "Visualizing-Filters-and-Feature-Maps-in-Convolutional-Neural-Networks-using-PyTorch",
+        "display": "Visualize Filters and Feature maps using PyTorch",
+        "description": "A way to visualize filters and feature maps in CNN",
+        "author": "LightMonarch",
+        "category": "machine learning",
+        "level": "basic",
+        "site": "",
+        "tags": "python, machine learning"
```
Comment (ming-tsai) [inline data/projects.json:110]:
```suggestion
        "tags": "python, machine learning, feature-maps-and-filter-visualizer"
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]

File: data/ideas.json (hunk 1 above)
Comment (Ashborn-SM) [reply data/ideas.json:34]: Not yet

**Review by Ashborn-SM — COMMENTED** [review]
Done the changes @ming-tsai

File: data/projects.json [hunk 4]
```diff
@@ -98,5 +98,15 @@
...
+    },
+    {
+        "name": "Visualizing-Filters-and-Feature-Maps-in-Convolutional-Neural-Networks-using-PyTorch",
+        "display": "Visualize Filters and Feature maps using PyTorch",
+        "description": "A way to visualize filters and feature maps",
+        "author": "Ashborn-SM",
+        "category": "machine learning",
+        "level": "basic",
+        "site": "",
+        "tags": "feature-maps-and-filter-visualizer, machine learning"
```
Comment (ming-tsai) [inline data/projects.json:110]:
```suggestion
        "tags": "feature-maps-and-filter-visualizer, machine learning, python"
```

**Review by ming-tsai — APPROVED** [review]
Nice work

**Pull request merged**